Sources/MenuContainerViewController.swift

```diff
@@ -90,14 +90,6 @@ open class MenuContainerViewController: UIViewController {
         )
 
         navigationMenuTransitionDelegate = MenuTransitioningDelegate(interactiveTransition: interactiveTransition)
-
-        let screenEdgePanRecognizer = UIScreenEdgePanGestureRecognizer(
-            target: navigationMenuTransitionDelegate.interactiveTransition,
-            action: #selector(MenuInteractiveTransition.handlePanPresentation(recognizer:))
-        )
-
-        screenEdgePanRecognizer.edges = .left
-        view.addGestureRecognizer(screenEdgePanRecognizer)
```

Comment:
ja bym to jednak dodał jako opcja a nie usuwał całkowicie, może jakiś bool przy inicjacji albo poperty?